Floofbot/Modules/Logging.cs

```diff
@@ -127,48 +102,31 @@ public async Task Toggle()
                 try
                 {
                     // check the status of logger
-                    string sqlCheckStatus = $"SELECT * FROM Logger WHERE ServerID = {Context.Guild.Id} LIMIT 1"; // check state
-                    using (SqliteCommand command = new SqliteCommand(sqlCheckStatus, dbConnection))
+                    var ServerConfig = _floofDB.LogConfigs.Find(Context.Guild.Id);
+
+                    bool bEnabled = ServerConfig.IsOn;
+                    if (!bEnabled)
+                    {
+                        ServerConfig.IsOn = true;
+                        await Context.Channel.SendMessageAsync("Logger Enabled!");
+                    }
+                    else if (bEnabled)
+                    {
+                        ServerConfig.IsOn = false;
+                        await Context.Channel.SendMessageAsync("Logger Disabled!");
+                    }
+                    else // should never happen, but incase it does, reset the value

```

Comment:
Does VS not give you a warning about this? If its something that should never happen (which it certainly appears to be the case), I'd just remove it altogether.

Reply:
this is legacy code from before this was created as a bool, I will delete this